(DOCSP-26987): @realm/react-ify 'Specify a Primary Key' (#2448)
mongodb/docs-realm: source/sdk/react-native/realm-database/schemas/primary-key.txt

   
   A **primary key** is a property that uniquely identifies an
   object. Realm Database automatically :ref:`indexes <react-native-indexes>`
   primary key properties, which allows you to read and modify objects based
   on their primary key efficiently.
   
   If an object type has a primary key, then all objects of that type must
   include the primary key property with a unique value among objects of

>    the same type in a realm. An object type may have at most one primary

Reviewer: ```suggestion
   the same type in a realm. An object type can have only one primary
```
